### DISCUSSION TITLE
Quick Translation Update (June 2024)

### DISCUSSION THREAD

**Discussion by CodingWonders** [CodingWonders/DISMTools#136] [Announcements]
DISMTools currently supports 4 languages: English, Spanish, French, and Portuguese. Now, with version 0.5.1 in development, a new language will come. For this, we'll have to go to a European country with beautiful places such as Venice or **Rome**. That's right, we're going to Italy!

![DT051_ItalianPoster](https://github.com/CodingWonders/DISMTools/assets/101426328/049be117-dc21-45f9-a5dd-b5ae6d5630ab)

Even though, in its current state, Italian translations haven't been started yet, they will be very soon, so stay tuned.

PS: you can use the picture as a poster.

**Comment by abdullah-erturk** [comment]
I couldn't see any language files in the program

If a language file is created for localization, I can be interested in Turkish translation.

**Reply by CodingWonders** [reply]
Currently, language translations are done in the source code, but they will be moved to specific language files in the future. I will make a new discussion when the moment arrives.

**Comment by abdullah-erturk** [comment]
Why is my request ignored?
